docs/04-guides/on-demand/store-video-asset-on-ipfs/_tabs.mdx

```diff
@@ -0,0 +1,36 @@
+import TabbedPages from "@site/src/components/TabbedPages";
+
+<head>
+  <title>Store Video Asset on IPFS | Livepeer Studio Docs</title>
+  <meta
+    property="og:title"
+    content="Store Video Asset on IPFS | Livepeer Studio Docs"
+  />
+</head>
+
+# Store Video Asset on IPFS
+
+This guide provides you the instructions to store a video asset on IPFS from the [Livepeer Studio Dashboard](./dashboard) 
+or through the [Livepeer API](./api).
```

Comment:
I had a hard time finding the API guide, it didn't show up on the side bar for me nor did I see this first line discretely mentioning the existence of it. I personally mostly skip these intro sections unless there's something calling my attention like a "box", tabbed display, or something like that. I believe this might be the case for other users as well, especially since we have a couple of boilerplate in the beginning of most guides now, like technical preparation etc which is mostly skippable.

So could we make these alternate guides more evident somehow? If it's hard to add some different UI component in the docs, could be just making it an enumeration, or giving it bigger letters with `###` or something like that 🤔